cdap-api/src/main/java/co/cask/cdap/api/artifact/ArtifactId.java

```diff
@@ -49,11 +49,7 @@ public ArtifactScope getScope() {
 
   @Override
   public String toString() {
-    return "ArtifactId{" +
-      "name='" + name + '\'' +
-      ", version=" + version +
-      ", scope='" + scope + '\'' +
-      '}';
+    return String.format("%s-%s-%s", scope, name, version);
```

Comment:
i had it as ':' before. But because of this issue (https://issues.apache.org/jira/browse/HADOOP-3257) we can't localize files with ':' in the name.